README.md

```diff
@@ -1 +1,3 @@
-# eq-questionnaire-schemas
\ No newline at end of file
+# eq-questionnaire-schemas
+
+A repo for building questionnaire schemas for eq-questionnaire-runner
```

Comment:
The Readme is a little light. Maybe mention the `build` and `translation-templates` command. It should also mention running the validator for the `test-schemas` command.